code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -322,6 +367,17 @@
 	moduleselect_icon = "medical"
 	can_be_pushed = FALSE
 	hat_offset = 3
+	borg_skins = list(
+		"Default" = list(SKIN_ICON_STATE = "medical", SKIN_TRANSFORM = TRUE),
+		"Basic" = list(SKIN_ICON = 'icons/mob/robots_extra.dmi', SKIN_ICON_STATE = "robot_medical", SKIN_LIGHT_KEY = "robot"),
+		"Drone" = list(SKIN_ICON = 'icons/mob/robots_extra.dmi', SKIN_ICON_STATE = "drone_medical", SKIN_LIGHT_KEY = "drone_standard"),
+		"Heavy" = list(SKIN_ICON = 'icons/mob/robots_extra.dmi', SKIN_ICON_STATE = "heavy_medical"),
```

Comment:
if this isn't heavy from tf2 I'm gonna cry

Reply:
Nope, just a chunky borg.